internal/nist/drbg.go

```diff
@@ -0,0 +1,64 @@
+// Package nist implements helpers to generate NIST's Known Answer Tests (KATs).
+package nist
```

Comment:
Maybe move to ` circl/internal/drgb_aes`. could you point me to its specification.

Reply:
It's quite a shitty DRGB. The only reason we should include it in Circl is because the NIST submissions are using it (because they have to use a NIST approved DRGB.)  I deduced the specification from `randombytes.c` in Dilithium.

Reply:
Then, I am ok with having it duplicated in the test file (as before), since there is no other use of it other than testing dilithium and kyber.

Reply:
I presume we will have more NIST PQC candidates in Circl, no?